components/06-examples/accordion-table.hbs

```diff
@@ -45,7 +45,12 @@
         </button>
       </th>
       <td>Boom 23 kappen</td>
-      <td>Verzonden</td>
+      <td>
+        <button class="icon btn btn-link">
```

Comment:
Hier mist het type attribuut. Verder hoort de class `icon` hier ook niet.

Ik ben wel ff kwijt wat deze `icon` class ook alweer is, ik zag 'm in de voorbeeld markup van Wido ook al voorbij komen. Kun je dat uitzoeken?

Reply:
Dit is gekopieerde markup uit hetvoorbeeld van Wibo. Hoe zullen we hiermee omgaan?